src/test/ui/fragments/FragmentFormContainer.test.js

```diff
@@ -0,0 +1,10 @@
+import 'test/enzyme-init';
+
+import { mapDispatchToProps } from 'ui/fragments/FragmentFormContainer';
+
+
+it('verify that  onSubmit is defined by mapDispatchToProps', () => {
```

Comment:
Pleade add a describe block, so that it's clear what's tested in the test report